test/blueprints/Do.Test.Blueprints.Service.Test/Communication/MockCommunicationFeature.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Do.Communication;
+
+namespace Do.Test.Communication;
+
+public class MockCommunicationFeature : TestServiceSpec
+{
+    [Test]
+    public async Task Default_mock_behaviour_can_be_setup_for_given_client_type()
+    {
+        var client = MockMe.TheClient<Singleton>();
+
+        var response = await client.Send(new Request(UrlOrPath: string.Empty, Method: HttpMethod.Post));
+
+        response.ShouldNotBeNull();
+        ((string?)response.GetContentAsObject())?.ShouldBe("test result");
+    }
+
+    [Test]
+    public async Task Default_mock_behaviour_can_have_multiple_setups()
+    {
+        var client = MockMe.TheClient<Operation>();
+
+        var response1 = await client.Send(new Request(UrlOrPath: "path1", Method: HttpMethod.Post));
+        response1.ShouldNotBeNull();
+        ((string?)response1.GetContentAsObject())?.ShouldBe("path1 response");
+
+        var response2 = await client.Send(new Request(UrlOrPath: "path2", Method: HttpMethod.Post));
+        response2.ShouldNotBeNull();
+        ((string?)response2.GetContentAsObject())?.ShouldBe("path2 response");
```

Comment:
leave a short comment here to lead a reader where to find this mock setup in the project